src/handler.js

```diff
@@ -116,11 +116,11 @@ export class Handler {
         this.monitor.timer('handle-message.success');
       } catch(err) {
         this.monitor.timer('handle-message.failure');
-        console.log(`Error caught when processing message. ${err.message}. ${err.stack}`);
+        this.monitor.reportError(err);
```

Comment:
We probably should suppress some of the errors that might bubble up to this level to prevent a lot of sentry alerts.  Currently the validateTask method will throw an error if the task does not have treeherder information, or the task.extra.treeherder object does not pass schema validation. 

Looks like most of the errors reported are for tasks missing the treeherder config, which probably should just be silently ignored.